tests/test_helpers.py

```diff
@@ -21,9 +21,10 @@
       "-a","Kurt Godel", "MC Escher", "Johann Sebastian Bach","-c", "Bitcoin",
       "--other_agencies", "Flatland","-n", "this is a sample added proposal"],
      "20_anewproposal has been added in proposals\n"),
-    (["helper", "a_expense", "159.18", "google", "Payment to Google Cloud to pay for cloud services",
-      "-b", "g", "dmref15", "mrsec14", "-d", "2020-06-18"],
-     "Payment to google has been added to expenses\n"),
+    (["helper", "a_expense", "159.18", "google", "Payment to google cloud",
+      "-g", "mrsec14", "dmref15", "-y", "ashaaban", "-s", "submitted",
```

Comment:
also, to make it more readable as documentation, let's use the full names of the variables here rather than the shortcuts, so ``--status`` rather than ``-s`` etc.

Reply:
done